k8s/contiv-vpp/values.yaml

```diff
@@ -3,6 +3,7 @@ contiv:
   nodeToNodeTransport: "vxlan"  # possible values: "vxlan", "srv6", "nooverlay"
   useSRv6ForServices: false
   useTAPInterfaces: true
+  srv6ServiceIP4: false
```

Comment:
Do we really need this option? Wouldn't `useSRv6ForServices: true` be enough? In IPv4 setup, it would mean IPv4 services over SRv6, in IPv6 setup it would mean IPv6 services over SRv6

Reply:
I don't actually see this being used in the code :)

Reply:
Sorry, my fault. I forgot/miss remove this value.